src/relay/transforms/device_annotation.cc

```diff
@@ -320,191 +326,11 @@ class AnnotatationVisitor : private ExprVisitor {
   Map<Expr, Integer> annotations_;
 };
 
-/*
- * \brief Return device allocation map based on the post order traversed graph.
```

Comment:
Should we replace this with a similar updated comment?

Reply:
This is all dead now.